Add deep learning applications in heavy ion physics

lgpang opened the pull request:
I'd like to add my paper to the list, thanks a lot!

lgpang committed:
Add deep learning applications in heavy ion physics

matthewfeickert commented:
@lgpang looks good. Can you please:
* Link to the corresponding INSPIRE page instead of arXiv
* Adopt the same date formatting as shown in the rest of the papers section
* Add the bibtex reference (as it appears on INSPIRE) to the HEPML.bib
?

lgpang committed:
Modify deep learning in heavy ion physics

Use inspire link instead of arxiv link;
Fixed date format

lgpang committed:
Add bibitem for deep learning in heavy ion

Add one bibitem for the equation-of-state-meter in heavy ion collisions using deep learning

matthewfeickert commented:
@lgpang Looks great. Many thanks for your paper and your contribution to the repo!
